### PR TITLE
Added section 4 - Tag types

### PR DESCRIPTION
**Pull request by cshaa**
Please check for mistakes.

### CODE REVIEW

**Comment by OscarGodson** [comment]
Seems ok, but can you run the build.sh file so the HTML doc is updated?

**Comment by cshaa** [comment]
I tried to but seems not to work. Should I launch it in the terminal or graphically?

**Comment by OscarGodson** [comment]
In your terminal you should be able to just run `./build.sh`. If that doesn't work try:

```
chmod u+x build.sh; ./build.sh
```

Still no luck I can do it on my end :)

**Comment by cshaa** [comment]
From my terminal:

```
michal@michal-notebook ~/Dropbox/Developing/HTML6 $ sudo ./build.sh
./build.sh: line 7: marked: command not found
 √ Build complete
michal@michal-notebook ~/Dropbox/Developing/HTML6 $ git status
# On branch master
# Your branch is ahead of 'origin/master' by 1 commit.
#
nothing to commit (working directory clean)
```

I cannot use paid mac app on my linux! xD
Don't worry, I'll do it my way.

**Comment by OscarGodson** [comment]
Paid mac app? That's just a shell script? The reason nothing was built is because you don't have marked installed: https://github.com/chjj/marked

**Comment by OscarGodson** [comment]
Your commit wont work because the code tags are a different format. It should be `<pre><code>` not `<p><code>`.

**Comment by cshaa** [comment]
When I was searching, I found Mac app. Ok, I'll try once again.

**Comment by OscarGodson** [comment]
Oh, ha: http://markedapp.com/

Funny there happens to be a Mac app called Marked for Markdown :P

**Comment by cshaa** [comment]
I give up. Don't know how to install marked, `npm install marked` doesn't work for me, command marked is still not found. Please, can you run it on your own? Thanks

**Comment by cshaa** [comment]
One last try :D

**Comment by OscarGodson** [comment]
Looks good, but can you remove the `<js:>` tags? This pull request was just for the tag types :)

File: README.md [hunk 1]
```diff
@@ -465,13 +465,32 @@ _Example:_
...
+```
+
+Unlike double tag, single doesn't need to be closed. Double tags usually have some text content so they are made of an opening and closing tag. If it has no text content, it can be shortened to the self-closing single variant. Examples:
+```xml
+<html:link href="./a.html">Text content</html:link>
+
+<!-- This shortand... -->
+<js:script src="./a.js" />
+<!-- ...means in fact this: -->
+<js:script src="./a.js"></js:script>
```
Comment (OscarGodson) [inline README.md]: We don't have this tag, but we can talk about it more in that ticket you opened, so could you replace with maybe any example that either has a custom tag like `<foo />` actually means `<foo></foo>` or a tag like `html:media`?

**Comment by OscarGodson** [comment]
Im glad you finally got it working! :+1:

File: README.md (hunk 1 above)
Comment (cshaa) [inline README.md]: OK, no problem.

**Comment by cshaa** [comment]
Is it right now?

**Comment by OscarGodson** [comment]
Yep!

**Pull request merged**